addon/components/cfb-form-editor/question.js

```diff
@@ -56,16 +56,19 @@ export default Component.extend(ComponentQueryManager, {
    * This uses manual DOM manipulation to avoid adding a single-use component.
    */
   addSlug() {
-    const namespace = this.options.get("namespace");
     const input = this.element.querySelector('[name="slug"]');
 
-    if (namespace && input) {
+    if (
+      this.namespace.length &&
```

Comment:
Why `.length`?

Reply:
To check if the namespace is not empty. I usually write `.length > 0` but tried to be as concise as possible.

Reply:
Since empty strings are falsy, you can drop `.length` alltogether.

Reply:
And with the patch discussed below `namespace` would even be `null`.